### ISSUE TITLE
bug(youtube/custom-branding): ReVanced specific splash screen does not show up on devices with Android 11 or older

### ISSUE THREAD

**Issue by xDARKxDEVILx** [ReVanced/revanced-patches#584]
### Type

Cosmetic

### Bug description

Using YouTube version 18.23.35 and latest patches version and custom branding patch, the splash screen logo is the YouTube logo instead of the new ReVanced logo

### Steps to reproduce

1- using YouTube version 18.23.35
2- using patches version 2.184.0
3- using custom branding patch and most of the default patches 
4- patch the apk successfully
5- the splash screen logo is not changed 

### Relevant log output

```shell
None
```

### Additional context

Android 11
MIUI 12.1
Xiaomi mi 9t

Please don't close and ask to edit with what more info is needed 

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by n1cKz1** [comment]
Now that YouTube is seemingly done rolling out the new splash screen globally can this be revisited?

**Comment by oSumAtrIX** [comment]
YouTube replaced it with an animation instead of an icon, therefor this issue and the respective code to change the splash icon lost their purpose. On older devices, YouTube may show the splash icon though.

**Comment by n1cKz1** [comment]
Idk if it's the case for everyone, but for me it still shows the icon for like half a second before the animation is played

**Comment by oSumAtrIX** [comment]
That's part of the animation.

**Comment by xDARKxDEVILx** [comment]
> Since the startup splash screen is no longer static and is now animated, this issue may not be relevant anymore.

Yeah, i think i will just close this issue, i upgraded to android 14 a while ago but kept it for other people benefit, you can close it whenever
